### PR TITLE
doc: add QEMU howto

### PR DESCRIPTION
**Pull request by rojkov**
[skip-ci]

### CODE REVIEW

File: doc/howtos/qemu.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+##########
+
+A Refkit wic image can be tested with QEMU as is. For that make sure that the
+`ovmf` target has been built::
+
+  $ bitbake ovmf
+
+Then use the command `runqemu` to start a QEMU virtual machine::
+
+  $ runqemu runqemu ovmf refkit-image-common wic
```
Comment (mythi) [inline doc/howtos/qemu.rst]: copy-paste error with all ```$ runqemu``` examples it seems: runqemu is written twice.

**Comment by rojkov** [comment]
> +
> +  $ runqemu runqemu ovmf refkit-image-common wic
>
> copy-paste error with all $ runqemu examples it seems: runqemu is written
> twice.
>
>
Right, too much thoughtless yanking in vim. Fixed.

**Comment by pohly** [comment]
Performance under QEMU is better with KVM enabled.

This can be done with:

    BB_ENV_EXTRAWHITE="$BB_ENV_EXTRAWHITE QEMU_USE_KVM" QEMU_USE_KVM=intel-corei7-64 {oe-selftest|runqem} ...

Or it can go into local.conf.sample, similar to this entry in refkit-ci.inc that I am currently testing in PR #97:

    # If we can access /dev/kvm, then let's use it.
    QEMU_USE_KVM = "${@ 'intel-corei7-64 intel-core2-32 qemux86 qemux86-64' if os.access('/dev/kvm', os.R_OK|os.W_OK) else '' }"

What do you guys think? This PR looks like the right place to add it.

**Comment by mythi** [comment]
I'd prefer the local.conf.sample option since it's easier from the user perspective.

OTOH, would that not be appropriate in the machine configs?

**Comment by pohly** [comment]
Choosing KVM changes the machine type. That's probably not something that should be done by default, i.e. not in the machine config.

**Comment by rojkov** [comment]
I'd second local.conf.sample. Will update the PR if there're no objections.

**Comment by mythi** [comment]
@rojkov can you rebase this and make it complete? We could start with ```QEMU_USE_KVM``` disabled by default (even if it's value is conditional).

**Comment by rojkov** [comment]
* rebased;
* added `QEMU_USE_KVM` to local.conf.sample (disabled by default);
* force pushed.

File: doc/howtos/qemu.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
+Using QEMU
+##########
+
+A Refkit wic image can be tested with QEMU as is. For that make sure that the
+`ovmf` target has been built::
+
+  $ bitbake ovmf
```
Comment (mythi) [inline doc/howtos/qemu.rst]: FWIW, meta-intel added ovmf in ```EXTRA_IMAGEDEPENDS``` so it should be available. But I guess this won't hurt.

**Pull request merged**